book/src/development/infrastructure/release.md

```diff
@@ -45,9 +44,8 @@ $ git push origin backport_remerge  # This can be pushed to your fork
 ```
 
 After this, open a PR to the master branch. In this PR, the commit hash of the
-`HEAD` of the `beta` branch must exists. In addition to that, no files should
-be changed by this PR.
-
+`HEAD` of the `beta` branch must exists. In addition to that, no files should be
+changed by this PR.
 
 ## Update the `beta` branch
```

Comment:
There is a section where we check something in the beta branch of rust. I think it would be better to state:

```
git fetch upstream
git checkout upstream/beta
BETA_SHA=$(git log --oneline -- src/tools/clippy/ | grep -o "Merge commit '[a-f0-9]*' into .*" | head -1 | sed -e "s/Merge commit '\([a-f0-9]*\)' into .*/\1/g")
```